_content/av/our-real-home_ajahn-chah

```diff
@@ -0,0 +1,21 @@
+---
+title: "Our Real Home"
+authors:
+  - "Ajahn Chah"
```

Comment:
🤔  This is the second by Ajahn Chah added to the library... would you like to make him an `_authors` entry? 😁